libfreerdp/core/gateway/rdg.c

```diff
@@ -40,6 +41,12 @@
 
 #define TAG FREERDP_TAG("core.gateway.rdg")
 
+#ifdef WITH_SPNEGO
```

Comment:
Should we refine that `#ifdef` ? Under windows we always have Negotiate, so I feel like `WITH_SPNEGO` is not the right test...